### PR TITLE
[Fluid] Embedded drag center

### PR DESCRIPTION
**Pull request by rubenzorrilla**
Now the embedded elements are capable to compute the point in where the drag resultant is applied by asking the `DRAG_FORCE_CENTER` variable to the `Calculate()`.

The `DragUtilities` have been also updated to do the total drag force center.

### CODE REVIEW

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp [hunk 1]
```diff
@@ -179,19 +188,49 @@ namespace Kratos {
...
+            bool is_embedded = true;
+
+            // Create a test element inside a modelpart
+            Model model;
+            ModelPart& model_part = model.CreateModelPart("Main", 3);
+            GenerateTestModelPart(model_part, is_embedded);
+
+            // Initialize the fluid element
+            for (auto& r_elem : model_part.Elements()) {
+                r_elem.Initialize();
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp]: please pass the processinfo

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp [hunk 2]
```diff
@@ -179,19 +188,49 @@ namespace Kratos {
             Model model;
             ModelPart& model_part = model.CreateModelPart("Main", 3);
             GenerateTestModelPart(model_part, is_embedded);
-            Element::Pointer p_element = model_part.pGetElement(1);
 
             // Initialize the fluid element
-            p_element->Initialize();
+            for (auto& r_elem : model_part.Elements()) {
+                r_elem.Initialize();
```
Comment (philbucher) [inline applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp]: same

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp (hunk 1 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp]: Done.

File: applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp (hunk 2 above)
Comment (rubenzorrilla) [reply applications/FluidDynamicsApplication/tests/cpp_tests/test_drag_utils.cpp]: Done. Thanks!

**Comment by rubenzorrilla** [comment]
@akodakkal

**Review by akodakkal — APPROVED** [review]

**Pull request merged**